### PR TITLE
Update home page content

### PR DESCRIPTION
**Pull request by rmzelle**
@adam3smith, can you review?

### CODE REVIEW

File: _pages/home.md [hunk 1]
```diff
@@ -87,7 +87,14 @@ multicolumn_list:
     url: https://www.zotero.org/
 ---
 
-Visit our [Authors](/authors/) section to learn how to obtain and use our styles; [Publishers](/publishers/) if you are a publisher or journal editor with an interest in CSL; [Developers](/developers/) if you want to edit or create styles, are looking to support CSL in your own software, or wish to contribute to development of CSL itself; [Blog](/blog/) for big news updates; [About us](/contact/) to see who is behind this open source project; and [Contact](/contact/) if you’re looking for support or just want to say hi.
+This website is divided into several sections:
 
-Our crowdsourced repository offers over 8000 free CSL citation styles, and CSL is used by commercial and open source products alike, including:
+* [Authors](/authors/) - learn how to obtain and use our CSL citation styles
```
Comment (adam3smith) [inline _pages/home.md]: how about "find and use" -- obtain sounds odd to me.

File: _pages/home.md [hunk 2]
```diff
@@ -87,7 +87,14 @@ multicolumn_list:
     url: https://www.zotero.org/
 ---
 
-Visit our [Authors](/authors/) section to learn how to obtain and use our styles; [Publishers](/publishers/) if you are a publisher or journal editor with an interest in CSL; [Developers](/developers/) if you want to edit or create styles, are looking to support CSL in your own software, or wish to contribute to development of CSL itself; [Blog](/blog/) for big news updates; [About us](/contact/) to see who is behind this open source project; and [Contact](/contact/) if you’re looking for support or just want to say hi.
+This website is divided into several sections:
```
Comment (adam3smith) [inline _pages/home.md]: Do we need this? It sounds so formal. Wouldn't just the list do?

File: _pages/home.md (hunk 1 above)
Comment (adam3smith) [reply _pages/home.md]: Would have to look, but might work better to also bold the category. 
Also, I think these should be endashes (`&endash;`) not hyphens.

**Review by adam3smith — COMMENTED** [review]
Looks good. A couple of suggestions

File: _includes/page__hero.html [hunk 3]
```diff
@@ -27,13 +27,6 @@
 >
   {% if page.header.overlay_color or page.header.overlay_image %}
     <div class="wrapper">
-      <h1 class="page__title" itemprop="headline">
-        {% if paginator %}
-          {{ site.title }}{% unless paginator.page == 1 %} {{ site.data.ui-text[site.locale].page | default: "Page" }} {{ paginator.page }}{% endunless %}
-        {% else %}
-          {{ page.title | default: site.title | markdownify | remove: "<p>" | remove: "</p>" }}
-        {% endif %}
-      </h1>
```
Comment (rmzelle) [inline _includes/page__hero.html]: @pmelchor, do you know from the top of your head if it's okay to change this template file? I want to suppress the page title on the homepage.

**Comment by rmzelle** [comment]
> Looks good. A couple of suggestions

Thanks! Made a few more edits, and done for now. Feel free to merge if you're happy with this.

**Comment by rmzelle** [comment]
(and squash please)

File: _includes/page__hero.html (hunk 3 above)
Comment (pmelchor) [reply _includes/page__hero.html]: As a best practice you want to keep a visible and unique `<h1>` main heading on each page. This is relevant for SEO and other markup related purposes.

If you don't like the design we can tweak that and/or find a different placing for the `<h1>` on the homepage.

In terms of the template, I think that right now we are only using that one for the homepage.

Comment (rmzelle) [reply _includes/page__hero.html]: Okay, I'll put it back :).

**Pull request merged**